### PR TITLE
Translation changes in file

### PR DESCRIPTION
**Pull request by chaitanyakole**

### CODE REVIEW

File: apps/front-end/src/pages/front-end/facilitator/edit/ArrayForm.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ export default function App({ userTokenInfo, footerLinks }) {
       ? "REFERENCE_DETAILS"
       : type === "experience"
       ? "JOB_EXPERIENCE"
-      : "VOLUNTEER_EXPERIENCE";
+      : "ADD_VOLUNTEER_EXPERIENCE";
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/facilitator/edit/ArrayForm.js:43]: it is working ??

File: apps/front-end/src/pages/front-end/facilitator/edit/arraySchema.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ export default {
       ],
       properties: {
         role_title: {
-          title: "JOB_TITLE",
+          title: "VOLUNTEER_TITLE",
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/facilitator/edit/arraySchema.js:15]: this is check and confirm with QA team

File: apps/front-end/src/pages/front-end/facilitator/edit/ArrayForm.js (hunk 1 above)
Comment (chaitanyakole) [reply apps/front-end/src/pages/front-end/facilitator/edit/ArrayForm.js:43]: yes it's working

File: apps/front-end/src/pages/front-end/facilitator/edit/arraySchema.js (hunk 2 above)
Comment (chaitanyakole) [reply apps/front-end/src/pages/front-end/facilitator/edit/arraySchema.js:15]: yes confirm

**Pull request merged**